1.Exploratory_Data_Analysis/scripts/0.nomic_cleanup_for_correlation.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pathlib
+
+import numpy as np
+import pandas as pd
+from sklearn.preprocessing import MinMaxScaler
+
+# In[7]:
+
+
+# Parameters
+cell_type = "SHSY5Y"
+
+
+# In[8]:
+
+
+# set import data paths
+nomic_df_path = pathlib.Path(
+    f"../2.Nomic_nELISA_Analysis/Data/clean/Plate2/nELISA_plate_430420_{cell_type}.csv"
+)
+
+nomic_df_filtered_out_path = pathlib.Path(
+    f"../2.Nomic_nELISA_Analysis/Data/clean/Plate2/nELISA_plate_430420_{cell_type}_cleanup4correlation.csv"
+)
+
+
+# In[9]:
+
+
+# read in data
+nomic_df_raw = pd.read_csv(nomic_df_path)
+
+
+# In[10]:
+
+
+# get the dimensions of the df
+print(f"{nomic_df_raw.shape}_before_filtering")
+# remove column if colname has pgml in it as we are using the Normalised values ('NSU') columns
+nomic_df = nomic_df_raw.loc[
+    :, ~nomic_df_raw.columns.str.contains("pgml", case=False, na=False)
+]
+print(f"{nomic_df.shape}_after_filtering")
+# if column does not contain [NSU] then prefix with Metadata_
+for col in nomic_df.columns:
+    if not any(x in col for x in ["NSU"]):
```

Comment:
Consider simplifying this if there's only one exclusion string (instead of using `any`).